apis/metastore/v1beta1/types.generated.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +generated:types
+// krm.group: metastore.cnrm.cloud.google.com
+// krm.version: v1beta1
+// proto.service: google.cloud.metastore.v1
+// resource: MetastoreBackup:Backup
+// resource: MetastoreService:Service
+// resource: MetastoreFederation:Federation
+
+package v1beta1
+
+import (
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/metastore/v1alpha1"
+)
+
+// +kcc:proto=google.cloud.metastore.v1.AuxiliaryVersionConfig
+type AuxiliaryVersionConfig struct {
```

Comment:
I would expect to see some of these types being removed from v1alpha1.  Are you using `generate.sh`?